package.json

```diff
@@ -21,6 +21,7 @@
         "generate-schema": "node .\\tools\\generate-3d-schema-types.js",
         "generate": "node ./tools/generate-component-template.js",
         "generate-legion": "node ./tools/generate-legion-template.js",
+        "generate-next": "node ./tools/generate-next-template.js",
```

Comment:
could we replace `npm run generate` to run this command and rename the existing command to `npm run generate-legacy`?